tests/test_matrixMap.py

```diff
@@ -90,6 +90,34 @@ def test_MatrixMapWithZeros(self):
         outdata2 = simplifiedMM.applyForward(indata)
         assert_allclose(outdata2, pred_outdata)
 
+    def test_MatrixMapShiftMapSimplify(self):
+        """Test that a non-square matrix map followed by a shift map can be simplified
+
+        This is ticket DM-10946
+        """
+        m1 = 1.0
+        m2 = 2.0
+        shift = 3.0
+        matrixMap = astshim.MatrixMap(np.array([[m1, m2]]))
+        self.assertEqual(matrixMap.nIn, 2)
+        self.assertEqual(matrixMap.nOut, 1)
+        shiftMap = astshim.ShiftMap([shift])
+        seriesMap = matrixMap.then(shiftMap)
+
+        indata = np.array([
+            [1.0, 2.0, 3.0],
+            [0.0, 1.0, 2.0],
+        ], dtype=float)
+        pred_outdata = np.array(m1 * indata[0] + m2 * indata[1] + shift)
+        pred_outdata.shape = (1, len(indata[0]))
```

Comment:
This seems really verbose. Why not just trust the unsimplified seriesMap (calling the `SeriesMap` constructor if needed to guarantee that it hasn't been simplified)?